auth-web/src/components/auth/account-settings/statement/Statements.vue

```diff
@@ -183,6 +207,11 @@ export default class Statements extends Mixins(AccountChangeMixin) {
     }
   }
 
+  get enableEFTPaymentMethod (): string | null {
+    const enableEFTPaymentMethod: string = LaunchDarklyService.getFlag(LDFlags.EnableEFTPaymentMethod)
```

Comment:
it will be undefined? or null? if LD is blocked? I think undefined doesn't have reactivity? might be worth checking with ublock origin and adding launchdarkly.com to the filters

Reply:
Let me fix sbc-common-components - give me 10m